tests/test_mmlubranch.py

```diff
@@ -0,0 +1,27 @@
+# Standard
+import os
+
+# First Party
+from instructlab.eval.mmlu import MMLUBranchEvaluator
+
+
+def test1():
+    print("===> Executing test1...")
+    try:
+        model_path = "instructlab/granite-7b-lab"
+        sdg_path = f"{os.path.dirname(os.path.realpath(__file__))}/testdata/sdg"
+        tasks = ["mmlu_pr"]
+        mmlu = MMLUBranchEvaluator(
+            model_path=model_path, sdg_path=sdg_path, tasks=tasks
+        )
+        overall_score, individual_scores = mmlu.run()
+        print(overall_score)
+        print(individual_scores)
+    except Exception as exc:
+        print(f"test1 failed: {exc}")
+        return False
+    return True
+
+
+if __name__ == "__main__":
```

Comment:
Is having a test-runner like pytest on the roadmap at some point? Not pressuring, purely curious.

Reply:
Yes I'd love to make these proper unit tests at some point!